pkg/util/prompt.go

```diff
@@ -28,6 +28,24 @@ func GetInput(defaultValue string) string {
 	return value
 }
 
+// GetQuotedInput reads input from an input buffer in single or double quotes and returns the result as a string.
+func GetQuotedInput(defaultValue string) string {
+	input := GetInput(defaultValue)
+
+	// Remove surrounding quotes, but only the closest ones.
+	if len(input) >= 2 && (input[0] == '\'' && input[len(input)-1] == '\'' || input[0] == '"' && input[len(input)-1] == '"') {
```

Comment:
I still wish we could use strings.Trim() :) 

Can't we do that now that we're expecting a filepath?

Reply:
Okay, I'm probably thinking too much here. I'll make it easier.